accord-core/src/test/java/accord/utils/Property.java

```diff
@@ -454,16 +460,35 @@ public <State, SystemUnderTest> void check(Commands<State, SystemUnderTest> comm
                                     throw new IllegalArgumentException("Unable to find next command");
                                 cmd = cmdGen.next(rs);
                             }
-                            history.add(cmd.detailed(state));
-                            Object stateResult = cmd.apply(state);
-                            cmd.checkPostconditions(state, stateResult,
-                                                    sut, cmd.run(sut));
+                            if (cmd instanceof MultistepCommand)
```

Comment:
Added for `TopologyMixupTest`.  Some times it's nice to produce a command that actually is multiple commands that run back to back.  This is mostly done to have a more detailed history more than anything else.

I am not a fan of the interface and finds that the sub command `details` isn't as nice as we compute them *before* running the first command, it would be nice to defer to right before executing the command...